moped-editor/src/views/projects/projectView/ProjectTeamTable.js

```diff
@@ -116,18 +148,24 @@ const ProjectTeamTable = ({
     {
       title: "Role",
       field: "role_id",
-      render: personnel => roles[personnel.role_id],
-      validate: rowData => !!rowData.role_id,
+      render: personnel => {
+        return personnel.role_id.map(chipRoleId => (
+          <Chip
+            className={classes.roleChip}
+            variant="outlined"
+            label={roles[chipRoleId]}
+          />
+        ));
+      },
+      // validate: rowData => Array.isArray(rowData) && rowData.length() > 0,
```

Comment:
I'm wondering how we can validate that a user added at least one role to a new project personnel entry. I tested this out, and it looks like this is handled by the upsert mutation so maybe that is the answer. 👍

Reply:
Oh man, good catch. I forgot to go back to this. Well, yes the mutation will fail if the data does not look good. But there will not be user feedback. Let me see if there is an obvious way to make this work.

Reply:
Lol, I wrote the rule once and uncommented, but it worked like charm just by uncommenting. 👍